src/app/page.tsx

```diff
@@ -8,14 +9,22 @@ const Home = () => {
   const [name, setName] = useState("");
   const [email, setEmail] = useState("");
 
-  const handleSubmit = (e) => {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     api({
       method: "POST",
       url: "/api/user",
       body: { name, email },
     })
-      .then(() => toast(`Submitted successfully!`))
+      .then((res) => {
+        if (res.status >= 200 && res.status < 300) {
```

Comment:
range checks dont work with http 😭 u can assume that if it goes thru its alr good

Reply:
oh i never wrote that, i think that was an issue with dev branch